src/crimson/common/exception.h

```diff
@@ -18,8 +18,21 @@ class system_shutdown_exception final : public std::exception{
   }
 };
 
+class crushmap_changed final : public std::exception{
```

Comment:
i don't think this is accurate. as the acting set is changed, for instance, when an osd is marked down. which is not tracked by the crush map. so probably, s/crushmap/actingset/ ?

Reply:
also, please add a space before "{".